ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,596 @@
+        bottomBarView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(bottomBarView)
+
+        NSLayoutConstraint.activate([
+            bottomBarView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            bottomBarView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -Constants.BottomBar.bottomOffset)
+        ])
+
+        self.bottomBarView = bottomBarView
+        updateExpandability()
+    }
+
+    private func setupBottomSheetLayout() {
+        NSLayoutConstraint.deactivate(heroCommandWidthConstraints)
+        heroCommandStack.distribution = .fillEqually
+
+        let commandStackContainer = UIView()
+        commandStackContainer.addSubview(heroCommandStack)
+
+        let sheetController = BottomSheetController(contentView: makeBottomSheetContent(headerView: commandStackContainer, expandedContentView: tableView))
+        sheetController.hostedScrollView = tableView
+        sheetController.collapsedContentHeight = bottomSheetHeroStackHeight
+        sheetController.expandedHeightFraction = Constants.BottomSheet.expandedFraction
+
+        addChild(sheetController)
+        view.addSubview(sheetController.view)
+        sheetController.didMove(toParent: self)
+
+        // We need to keep a reference to this because the margin changes based on expandability
+        let heroStackTopConstraint = heroCommandStack.topAnchor.constraint(equalTo: commandStackContainer.topAnchor, constant: bottomSheetHeroStackTopMargin)
+        bottomSheetHeroStackTopConstraint = heroStackTopConstraint
+
+        NSLayoutConstraint.activate([
+            sheetController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            sheetController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            sheetController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            sheetController.view.topAnchor.constraint(equalTo: view.topAnchor),
+            heroCommandStack.leadingAnchor.constraint(equalTo: commandStackContainer.leadingAnchor, constant: Constants.BottomSheet.heroStackLeadingTrailingMargin),
+            heroCommandStack.trailingAnchor.constraint(equalTo: commandStackContainer.trailingAnchor, constant: -Constants.BottomSheet.heroStackLeadingTrailingMargin),
+            heroCommandStack.bottomAnchor.constraint(equalTo: commandStackContainer.bottomAnchor, constant: -Constants.BottomSheet.heroStackBottomMargin),
+            heroStackTopConstraint
+        ])
+
+        bottomSheetController = sheetController
+        updateExpandability()
+    }
+
+    private func makeBottomBarByEmbedding(contentView: UIView) -> UIView {
+        let bottomBarView = UIView()
+        bottomBarView.layer.shadowColor = Constants.BottomBar.Shadow.color
+        bottomBarView.layer.shadowOpacity = Constants.BottomBar.Shadow.opacity
+        bottomBarView.layer.shadowRadius = Constants.BottomBar.Shadow.radius
```

Comment:
Can avoid redundant method invocations under the hood and yield cleaner code overall by saving bottomBarView.layer to a local instead of fetching it for each call